### PR TITLE
Logstash persistent queue improvement

### PR DESCRIPTION
**Pull request by PMazarovich**
### Motivation and context

In current implementation of ELK there is no persistent queue in logstash. This can lead to the loss of important analytics data. With this improvement any data, that was sent by cvat will eventually be sent into logstash if logstash instance was unavaliable for some time. Or to elasticsearch instance if it was unavaliable. So, all of features of persistent queue is now ready for working with cvat.
As a side note, there is a metadata with year and month added. This is done for better flexibility in index management in elasticsearch.
### How has this been tested?

It was tested manually
### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by azhavoro** [comment]
@PMazarovich Could you please merge current develop changes (or rebase, as you prefer) to fix CI build?

**Comment by PMazarovich** [comment]
Rebase done

**Review by azhavoro — APPROVED** [review]

File: cvat/settings/base.py [hunk 1]
```diff
@@ -386,11 +394,16 @@ def add_ssh_keys():
...
             'level': 'INFO',
-            'class': 'logstash.TCPLogstashHandler',
+            'class': 'logstash_async.handler.AsynchronousLogstashHandler',
+            'formatter': 'logstash',
+            'transport': 'logstash_async.transport.HttpTransport',
+            'ssl_enable': False,
+            'ssl_verify': False,
             'host': os.getenv('DJANGO_LOG_SERVER_HOST', 'localhost'),
-            'port': os.getenv('DJANGO_LOG_SERVER_PORT', 5000),
+            'port': os.getenv('DJANGO_LOG_SERVER_PORT', 8080),
```
Comment (nmanovic) [inline cvat/settings/base.py:403]: What is the reason to change the port from 5000 to 8080?

Comment (PMazarovich) [reply cvat/settings/base.py:403]: For http protocol, port 8080 is standard one, and logstash now works via http because of persistent queue

Comment (nmanovic) [reply cvat/settings/base.py:403]: @azhavoro , we use 8080 for cvat_proxy. Can it be a problem?

Comment (azhavoro) [reply cvat/settings/base.py:403]: This is not open port, I don't see any problem here. We use the same port number for cvat_server https://github.com/openvinotoolkit/cvat/blob/develop/Dockerfile#L157

**Review by nmanovic — APPROVED** [review]

**Comment by nmanovic** [comment]
@PMazarovich , thanks for the contribution!

**Comment by chaoyifei** [comment]
HI, the log in the rq like INFO:cvat.server_33:Create Task #33 ,Uploadpath ...  is not into logstash

**Pull request merged**